WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsOnboardingError.swift

```diff
@@ -35,6 +35,7 @@ struct InPersonPaymentsOnboardingError: View {
             }
             if learnMore {
                 InPersonPaymentsLearnMore(viewModel: LearnMoreViewModel(tappedAnalyticEvent: learnMoreAnalyticEvent))
+                    .padding(.vertical, 8)
```

Comment:
I see we add this padding in multiple places, would it be convenient if we add it directly to the  InPersonPaymentsLearnMore view?

Reply:
No; I deliberately removed it from that view so that we could omit it in some places. 

I don't think views should necessarily have external padding, because it makes them less flexible for use in multiple places. It could be passed as a parameter, but that didn't seem particularly idiomatic.